packages/request-node/src/server.ts

```diff
@@ -3,22 +3,36 @@ import { Logger } from './logger';
 import withShutdown from 'http-shutdown';
 import { RequestNode } from './requestNode';
 import { getDataAccess } from './dataAccess';
-import KeyvFile from 'keyv-file';
 import { getDataStorage } from './dataStorage';
+import ConfirmedTransactionStore from './request/confirmedTransactionStore';
+import { EvmChains } from '@requestnetwork/currency';
+import { getEthereumStorageNetworkNameFromId } from '@requestnetwork/ethereum-storage';
+import { SubgraphClient } from '@requestnetwork/thegraph-data-access';
 
 // Initialize the node logger
 const logger = new Logger(config.getLogLevel(), config.getLogMode());
 
+const getNetwork = () => {
+  const network = getEthereumStorageNetworkNameFromId(config.getStorageNetworkId()) as any;
+  if (!network) {
+    throw new Error(`Storage network not supported: ${config.getStorageNetworkId()}`);
+  }
+  EvmChains.assertChainSupported(network);
+  return network;
+};
+
 export const getRequestNode = (): RequestNode => {
-  const initializationStoragePath = config.getInitializationStorageFilePath();
-  const store = initializationStoragePath
-    ? new KeyvFile({
-        filename: initializationStoragePath,
-      })
-    : undefined;
+  const network = getNetwork();
   const storage = getDataStorage(logger);
-  const dataAccess = getDataAccess(storage, logger);
-  return new RequestNode(dataAccess, storage, store, logger);
+  const dataAccess = getDataAccess(network, storage, logger);
+
+  // the confirmed transaction feature is specific to usage with a Request Node, so isn't exposed by dataAccess.
```

Comment:
What does this comment mean? Is there a way to make it more clear?

Reply:
it means that this confirmation flow doesn't really matter outside of the usage with Client and Node, so it doesn't fit in the underlying packages / interfaces.

is this better? 
 ```suggestion
  // we access the subgraph client directly, not through the data access, because this feature is specific to RN use with Request Node. Without a node, the confirmation process would be different, so this doesn't fit in the data access layer
```

Reply:
Yes. this is more clear. OK to commit :+1: